service/src/main/java/de/starwit/service/impl/PostFlywayService.java

```diff
@@ -0,0 +1,149 @@
+            LOG.info("Importing sample data structure from folder " + scenarioImportFolder);
+            if (importMitigationTypes(scenarioImportFolder)) {
+                LOG.info("Import of mitigation types successful, importing incident types");
+                if (importIncidentTypes(scenarioImportFolder)) {
+                    importDemoData(scenarioImportFolder);
+                }
+            }
+        }
+    }
+
+    private boolean importMitigationTypes(String folder) {
+        if (mtRepository.findAll().size() > 0) {
+            LOG.info("Mitigation types already imported. Skipping import.");
+        } else {
+            File file = new File(folder + "/" + mititgationTypeFileName);
+            if (file.exists()) {
+                LOG.info("Importing mitigation types from file " + file.getAbsolutePath());
+                try {
+                    List<MitigationActionTypeEntity> mitigationTypes = mapper.readValue(file,
+                            new TypeReference<List<MitigationActionTypeEntity>>() {
+                            });
+                    mtRepository.saveAll(mitigationTypes);
+                } catch (IOException e) {
+                    LOG.error("Can't parse mitigation types, aborting import " + e.getMessage());
+                }
+                return true;
+            } else {
+                LOG.warn("Mitigation types file not found. Skipping import.");
+            }
+        }
+        return false;
+    }
+
+    private boolean importIncidentTypes(String folder) {
+        if (ttRepository.findAll().size() > 0) {
+            LOG.info("Incident types already imported. Skipping import.");
+        } else {
+            File file = new File(folder + "/" + incidentTypeFileName);
+            if (file.exists()) {
+                LOG.info("Importing Incident types from file " + file.getAbsolutePath());
+                try {
+                    List<TrafficIncidentTypeEntity> trafficIncidentTypes = mapper.readValue(file,
+                            new TypeReference<List<TrafficIncidentTypeEntity>>() {
+                            });
+                    ttRepository.saveAll(trafficIncidentTypes);
+                } catch (IOException e) {
+                    LOG.error("Can't parse Incident types, aborting import " + e.getMessage());
+                }
+                return true;
+            } else {
+                LOG.warn("Incident types file not found. Skipping import.");
+            }
+        }
+        return false;
+    }
+
+    private boolean importDemoData(String folder) {
+        File file = new File(folder + "/" + demoDataFileName);
+        if (file.exists()) {
+            LOG.info("Importing demo data from file " + file.getAbsolutePath());
+            try {
+                Path path = Path.of(folder + "/" + demoDataFileName);
+                String content = Files.readString(path, StandardCharsets.UTF_8);
+                int timeOffset = 13;
```

Comment:
I don't understand the handling of the timeOffset

Reply:
added comment